views/components/GraphTable/index.jsx

```diff
@@ -6,21 +6,41 @@ import {
   TableRow,
   Table,
   Paper,
+  Toolbar,
+  Button,
 } from '@material-ui/core'
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown'
 import {
   graphTableRowDataBySite,
   formatGraphTableCellData,
   graphTableColumns,
 } from './helpers'
+import { fetchGraphTableCSV } from '../../fe-utils/fetchUtil'
 
-const GraphTable = ({ graph }) => {
+const GraphTable = ({ graph, classes }) => {
   if (!graph) return null
 
   const tableColumns = graphTableColumns(graph.labels)
   const tableRows = graphTableRowDataBySite(graph.dataBySite)
```

Comment:
are these helpers now duplicated on the server and the client?

Reply:
Yes, we had a discussion about this and my anxiety to keep things DRY. 
https://gnardog.slack.com/archives/D02E4F85S6T/p1681237219131309

Reply:
and wanting to update the controller object so that we do this processing in one place instead of the two.

Reply:
ok. I responded in Slack but for posterity:

we shouldn't be duplicating those helper methods we should be either doing the work on the server or the client - If we're doing the csv work on the server then let's send back the chart table data in the response for the table to use and remove those duplicated helpers.